courses/machine_learning/intro_to_tensorflow/3_keras_sequential_api.ipynb

```diff
@@ -0,0 +1,482 @@
+{
```

Comment:
<p>To have very simple &amp; readable model code, I'd import all the needed layers and models at the beginning where all the other imports are made:</p><p> </p><p> </p><p><code>from tensorflow.kears.models import Sequential</code></p><p><code>from tensorflow.keras.layers import (</code></p><p><code>    DenseFeature,</code></p><p><code>    Dense,</code></p><p><code>)</code></p><p> </p><p> </p><p>Then I'd define the model directly by passing the list of layers to the "Sequential" ; this would look like the following:</p><p> </p><p><code>model = Sequential([</code></p><p><code>    DenseFeatures(feature_columns=feature_columns.values()),</code></p><p><code>    Dense(units=32, activation="relu", name="h1") ,</code></p><p><code>    Dense(units=8, activation="relu", name="h2"),</code></p><p><code>    Dense(units=1, activation="linear", name="output")</code></p><p><code>])</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/559/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
will do